ansible/roles/test/tasks/dir_bcast.yml

```diff
@@ -0,0 +1,42 @@
+#-----------------------------------------
+# Run dir_bcast test and Perform log analysis.
+#-----------------------------------------
+
+# Pre-check testbed_type value
+- fail: msg="testbed_type is not defined."
+  when: testbed_type is not defined
+
+- fail: msg="testbed_type {{testbed_type}} is invalid."
```

Comment:
Please make spacing after `{{` and before `}}` consistent throughout file. Some occurrences have no space, others have one.

Reply:
Removed the spacing as in few other files and made it consistent